climpred/tests/test_perfect_model_prediction.py

```diff
@@ -34,25 +34,6 @@ def PM_ds_control1d():
     return ds
 
 
-@pytest.mark.parametrize('metric', ('rmse', 'pearson_r'))
```

Comment:
What’s wrong with this test? It made sure that init ensembles have low p values for the first lead independent of the metric used.

Reply:
Good catch, I have no idea why it was removed.